front/app/containers/Admin/dashboard/components/ChartFilters/TopicFilter.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
This used to be in `front/app/containers/Admin/dashboard/components/ChartFilters.tsx`, and the `generateTopicOptions` logic used to be in `front/app/containers/Admin/dashboard/summary/index.tsx`.